qiskit/providers/ibmq/managed/managedjobset.py

```diff
@@ -326,17 +337,18 @@ def error_messages(self) -> Optional[str]:
 
     @requires_submit
     def cancel(self) -> None:
-        """Cancel all managed jobs."""
+        """Cancel all jobs in this job set."""
         for mjob in self._managed_jobs:
             mjob.cancel()
 
     @requires_submit
     def jobs(self) -> List[Union[IBMQJob, None]]:
-        """Return a list of submitted jobs.
+        """Return jobs in this job set.
 
         Returns:
-            A list of IBMQJob instances that represents the submitted jobs. The
-                entry is ``None`` if the job submit failed.
+            A list of :class:`IBMQJob<qiskit.providers.ibmq.job.IBMQJob>`
+            instances that represents the submitted jobs.
+            An entry is ``None`` if the job failed to be submitted.
```

Comment:
Might be good to specify "in the list", so it would read as: "An entry in the list is ``None`` if the job failed to be submitted."